### ISSUE TITLE
allNeighborsIndex returning nothing

### ISSUE THREAD

**Issue by oculardexterity** [jacomyal/sigma.js#747]
`sigma.classes.graph.addMethod('neighbors', function(nodeId) {
    var k,
        neighbors = {},
        index = this.allNeighborsIndex[nodeId] || {};

    for (k in index) {
        neighbors[k] = this.nodesIndex[k];
    }
    return neighbors;
});`

Hi,
I am implementing the above code (adding neighbors function to the graph). Is there an obvious reason why this.allNeighborsIndex[nodeId] will not return anything? I've tried with numerous nodes (to make sure they do have edges), but none return anything.

I can verify that the node is found by this.allNeighborsIndex by calling this.allNeighborsIndex.has(nodeId) — but still it apparently has no neighbors!

I am adding nodes one by one — is that an issue? Do I need to re-add this function after building the graph?

**Comment by AlexAlexGoTO** [comment]
same problem

**Comment by Yomguithereal** [comment]
Does the problem still exists?

**Comment by hejiheji001** [comment]
Use http://sigmajs.org/assets/js/sigma.min.js

**Comment by danielritter** [comment]
Yes, I have the same problem. I get nothing. When I use the file above sigma.min.js I get an javascript error in the sigma.classes.quad.js file. I use the linkurious 1.5.2 version.
